test/integration/targets/aws_caller_facts/tasks/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# ============================================================
+- name: test caller facts are returned
+  block:
```

Comment:
This block isn't really required (as we don't use any block features, such as `always`)